tests/unit/BlockStorageMultiSig.d

```diff
@@ -92,7 +93,7 @@ private void main ()
         assert(block2.header.validators[1] == true, format!"block at height %s:\n%s\n\n"(h, prettify(block2)));
         block2.header.signature = SIG2;
         block2.header.validators[0] = true;
-        storage.updateBlockMultiSig(block2);
+        storage.updateBlockSig(Height(h), block2.hashFull(), SIG2, block2.header.validators);
```

Comment:
This test fails now with `Hash of block to update differs from the expected one`